packages/opentelemetry/src/index.ts

```diff
@@ -36,7 +36,12 @@ export { setOpenTelemetryContextAsyncContextStrategy } from './asyncContextStrat
 export { wrapContextManagerClass } from './contextManager';
 export { SentryPropagator } from './propagator';
 export { SentrySpanProcessor } from './spanProcessor';
-export { SentrySampler } from './sampler';
+export {
+  SentrySampler,
+  sentrySamplerNoDecision,
+  sentrySamplerNotSampled,
+  sentrySamplerSampled,
```

Comment:
m: Couldn't we expose a single method that takes in `SamplingDecision` from `@opentelemetry/sdk-trace-base` or similar?

```
sentrySampler({
  decision, // SamplingDecision enum
  context,
  spanAttributes,
})
```

It feels like a lot of overhead for us to export three different methods, and to me it feels better to expose the underlying enum being used.